jwt.go

```diff
@@ -0,0 +1,67 @@
+package clerk
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/go-jose/go-jose/v3/jwt"
+)
+
+type key string
+
+const activeSessionClaims = key("activeSessionClaims")
```

Comment:
💡 We can prefix/namespace the context key with `clerk` so the scope is even clearer if someone inspects its contents.

Reply:
Great idea! Updated.